Setting/Extensions/extenstion+viewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  extenstion+viewController.swift
+//  Setting
+//
+//  Created by ULADZISLAU ZAKREUSKI on 28.08.22.
+//
+
+import UIKit
+
+extension CustomViewController: UITableViewDelegate, UITableViewDataSource {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return items.count
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return items[section].cell.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: CustomTableViewCell.identifier, for: indexPath) as? CustomTableViewCell else { return UITableViewCell() }
+        return cell
+    }
+    
+    func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        guard let cell = cell as? CustomTableViewCell else { return }
+        let model = items[indexPath.section].cell[indexPath.row]
+        cell.configure(with: model)
+    }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let viewController = DetailViewController()
+        viewController.itemCell = items[indexPath.section].cell[indexPath.row]
```

Comment:
itemCell лучше делать приватным свойством и передавать через инициализатор, чтобы нельзя было изменить его из вне. Для каждый ячейки DetailView создается свой, поэтому нет смысла пытаться менять в нем содержимое. Так в памяти находяться совершенно разные DetailViewController
```suggestion
        let itemCell =  items[indexPath.section].cell[indexPath.row]
        let viewController = DetailViewController(item: itemCell)
```